Plugin.NetStandardStorage/Implementations/Folder.cs

```diff
@@ -258,5 +259,18 @@ public void Delete()
                 Directory.Delete(this.FullPath);
             }
         }
+
+        public void DeleteWithContent()
+        {
+            if (!this._canDelete)
+            {
+                throw new IOException("The root folder can't be deleted !");
+            }
+
+            if (this.CheckFolderExists(this.FullPath))
+            {
+                Directory.Delete(this.FullPath, true);
```

Comment:
please specify what `true` means by specifying the `recursive` parameter name, like this: `Directory.Delete(this.FullPath, recursive: true);`